package.json

```diff
@@ -72,12 +72,12 @@
     "eslint-config-react": "^1.1.7",
     "eslint-plugin-prettier": "^4.0.0",
     "gh-pages": "^3.2.3",
+    "husky": "^7.0.0",
     "less": "^4.1.2",
     "less-loader": "^7.3.0",
     "less-plugin-aliases": "^1.0.3",
-    "prettier": "^2.5.1",
+    "prettier": "^2.6.2",
```

Comment:
not sure why this updated but should be fine since there's no breaking changes

Reply:
Oops, I think accidentally installed prettier with npm inside the directory